ModList/UI/BSML/MarkdownTextHandler.cs

```diff
@@ -18,21 +15,46 @@ public class MarkdownTextHandler : TypeHandler<MarkdownText>
     {
         public override Dictionary<string, string[]> Props { get; } = new Dictionary<string, string[]>
         {
-            { "text", new[] { "text", "value" } },
-            { "childText", new[] { "_children" } },
-            { "linkPressed", new[] { "link-pressed" } },
-            { "linkColor", new[] { "link-color" } },
-            { "autolinkColor", new[] { "autolink-color" } },
+            {
+                "text", new[]
+                {
+                    "text",
+                    "value"
+                }
+            },
+            {
+                "childText", new[]
+                {
+                    "_children"
+                }
+            },
+            {
+                "linkPressed", new[]
+                {
+                    "link-pressed"
+                }
+            },
+            {
+                "linkColor", new[]
+                {
+                    "link-color"
+                }
+            },
+            {
+                "autolinkColor", new[]
+                {
+                    "autolink-color"
+                }
+            }
```

Comment:
These do not need to be exploded like they are. It is so much more readable when each one is on its own line.

Reply:
Will look into the .editorconfig file for this.